go/vt/vtgate/vindexes/lookup.go

```diff
@@ -104,6 +108,14 @@ func (ln *LookupNonUnique) Map(ctx context.Context, vcursor VCursor, ids []sqlty
 	return ln.MapResult(ids, results)
 }
 
+func (ln *LookupNonUnique) Lookup(ctx context.Context, vcursor VCursor, ids []sqltypes.Value) ([]*sqltypes.Result, error) {
+	results, err := ln.lkp.Lookup(ctx, vcursor, ids, vtgatepb.CommitOrder_NORMAL)
+	if err != nil {
+		return nil, err
+	}
+	return results, err
+}
+
```

Comment:
I think this is not used and hence can be removed.

Reply:
Ah, right, I must have committed this by accident! Thanks for catching this. 🙇‍♂️

Reply:
Done! This change was removed.